languages/ILSMessages/en.ini

```diff
@@ -0,0 +1,7 @@
+available_for_pickup = "Your requested material is available for pickup"
+checkout_block = "You can't checkout more items"
```

Comment:
"checkout" => "check out"?

Reply:
Yes, I agree, "check out" would be better.

Reply:
fixed